packages/snap/src/rpc/setBirthdayBlock.tsx

```diff
@@ -0,0 +1,48 @@
+import { Form, Box, Heading, Input, Button, Text } from "@metamask/snaps-sdk/jsx";
+
+type BirthdayBlockForm = {customBirthdayBlock: number | null}
+
+export async function setBirthdayBlock (): Promise<number | null> {
+
+  const interfaceId = await snap.request({
+    method: 'snap_createInterface',
+    params: {
+      ui:
+        <Form name="birthday-block-form">
+          <Box>
+            <Heading>Optional syncing block height</Heading>
+            <Text>
```

Comment:
It would also be nice to give a user a hint about the current block height as a reference point.

Reply:
Guess I would need "endowment:network-access" and some API providing current block of the chain. Or did I miss some method in wasm returning latest block without initialized wallet?

Reply:
you can pass it from front-end when calling setBirthdayBlock()